gtsfm/evaluation/compare_metrics.py

```diff
@@ -0,0 +1,66 @@
+"""Script to log metrics for comparison across SfM pipelines (GTSfM, COLMAP, etc.) as a JSON.
+
+Authors: Jon Womack
+"""
+import os
+from typing import Dict, List
+
+import gtsfm.utils.io as io_utils
+from gtsfm.evaluation.metrics import GtsfmMetric, GtsfmMetricsGroup
+
+import thirdparty.colmap.scripts.python.read_write_model as colmap_io
+
+
+def compare_metrics(
+    txt_metric_paths: Dict[str, str],
+    json_path: str,
+    GTSFM_MODULE_METRICS_FNAMES: List[str],
```

Comment:
is the documentation for this function up to date? I see the args section isn't but the docstring does not make a lot of sense either.

Reply:
Updated

Reply:
Thanks, I've addressed this.